### PR TITLE
Reference copy of the moderation indexer.

### PR DESCRIPTION
**Pull request by gabehamilton**
This is a reference copy of the moderation indexer. The running code at https://near.org/dataplatform.near/widget/QueryApi.App?selectedIndexerPath=dataplatform.near/moderation&view=editor-window is the source of truth.

### CODE REVIEW

**Comment by gabehamilton** [comment]
I intentionally didn't modularize it into many functions yet but will when I add the next features.

**Comment by roshaans** [comment]
can we put this indexer inside of a `moderation` folder?

**Comment by gabehamilton** [comment]
> can we put this indexer inside of a `moderation` folder, so later we can add the `feed` folder for the `feed-indexer`

Yes, I actually meant to do that. I'm adding the social_feed and verifications indexers in more PRs.

**Review by roshaans — APPROVED** [review]
LGTM

File: indexers/README.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# QueryApi Indexers used by components
+
+Several components use QueryApi Indexers as a source of data, specifically the ActivityFeed Posts & 'Following' Posts.
+
+These indexers can be modified at https://near.org/dataplatform.near/widget/QueryApi.App under the dataplatform.near account.
+
+Each
```
Comment (roshaans) [inline indexers/README.md:7]: nit: remove

```suggestion
```

**Pull request merged**